target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -67,63 +68,95 @@
 		};
 
 		usb {
-			gpio-export,name = "usb";
+			gpio-export,name = "usb_power_off";
 			gpio-export,output = <1>;
-			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+
+		// Used to enable power-over-ethernet passthrough to eth4 (according to OEM source).
+		poe_out {
+			gpio-export,name = "poe_passthrough";
+			gpio-export,output = <0>;
+			gpios = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+		};
+
+		poe_status {
+			gpio-export,name = "poe_status";
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
 		};
 	};
 };
 
 &spi0 {
 	status = "okay";
 
-	m25p80@0 {
+	w25q128@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <10000000>;
-		m25p,chunked-io = <32>;
+		// XXX matching RBM11/RBM33, untested
+		spi-max-frequency = <3125000>;
```

Comment:
I think 3.125MHz is way too slow for w25q128

Reply:
@dengqf6, I was browsing over the different SPI vedor specifications and dts code. I found no correlation between what I see in reference documentation and dts code. :-(

3Mhz is indeed way too slow for the w25q128 as per reference. It can do up to 133Mhz. However I have no idea whether it is safe to use this number.

@dengqf6, @mkresin, I would be happy to know your opinion here, before I increase the frequency.

thanks!

Reply:
I've no idea what the max spi freq of the mt7621 is.

Reply:
If unsure, leave it untouched

Reply:
@mkresin, @dengqf6, mt7621 datasheet says 20Mhz. I'll use it and give it some tests.